tests/e2e/specs/editor/media/insertMediaFromDialog.js

```diff
@@ -28,6 +28,8 @@ describe('Inserting Media from Dialog', () => {
 
     await expect(page).not.toMatchElement('[data-testid="FrameElement"]');
 
+    // TODO Use mediaUpload here.
```

Comment:
Can we do this in this PR?

Then I believe we only need `uploadMedia` everywhere, and `uploadFile` doesn't need to be exported as it's only used within `uploadMedia`.

Reply:
This is done the follow up PR. https://github.com/google/web-stories-wp/pull/5938 But I can merge that PR into this one, if help.

Reply:
I see. Let's do that then 👍

Reply:
Merged